caas/kubernetes/clientconfig/k8s_test.go

```diff
@@ -169,11 +170,11 @@ func (s *k8sConfigSuite) SetUpTest(c *gc.C) {
 	s.dir = c.MkDir()
 }
 
-// writeTempKubeConfig writes yaml to a temp file and sets the
+// writeTempKubeConfigAndSetEnv writes yaml to a temp file and sets the
 // KUBECONFIG environment variable so that the clientconfig code reads
 // it instead of the default.
 // The caller must close and remove the returned file.
-func (s *k8sConfigSuite) writeTempKubeConfig(c *gc.C, filename string, data string) (*os.File, error) {
+func (s *k8sConfigSuite) writeTempKubeConfigAndSetEnv(c *gc.C, filename string, data string) (*os.File, error) {
 	fullpath := filepath.Join(s.dir, filename)
```

Comment:
We can just call `writeKubeConfigFileToDir` right?

Reply:
yup will refactor this thanks!